tasklist/importer-860/src/main/java/io/camunda/tasklist/zeebeimport/v860/processors/common/UserTaskRecordToVariableEntityMapper.java

```diff
@@ -36,15 +40,19 @@ public List<TaskVariableEntity> mapVariables(final Record<UserTaskRecordValue> r
 
       final Map<String, Object> variablesMap = recordValue.getVariables();
       for (final Map.Entry<String, Object> varMap : variablesMap.entrySet()) {
-        final String varValue = String.valueOf(varMap.getValue());
-
+        final String varValue;
+        try {
+          varValue = objectMapper.writeValueAsString(varMap.getValue());
+        } catch (final JsonProcessingException e) {
+          LOGGER.error("Failed to parse variable %s".formatted(varMap), e);
+          throw new RuntimeException(e);
```

Comment:
Lets use the Tasklist Custom Exception here  
'TasklistRuntimeException'
over here